packages/config/config/devices/0x0371/zwa005.json

```diff
@@ -137,304 +135,106 @@
-					"value": 4
-				},
-				{
-					"label": "Pink",
-					"value": 5
-				},
-				{
-					"label": "Cyan",
-					"value": 6
-				},
-				{
-					"label": "Purple",
-					"value": 7
-				},
-				{
-					"label": "Orange",
-					"value": 8
-				}
-			]
-		},
-		"15": {
-			"label": "Led Color For Wakeup Notification Report",
-			"description": "LED color when sending a wakeup report",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 8,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disable",
-					"value": 0
-				},
-				{
-					"label": "Red",
-					"value": 1
-				},
-				{
-					"label": "Green",
-					"value": 2
-				},
-				{
-					"label": "Blue",
-					"value": 3
-				},
-				{
-					"label": "Yellow",
-					"value": 4
-				},
-				{
-					"label": "Pink",
-					"value": 5
-				},
-				{
-					"label": "Cyan",
-					"value": 6
-				},
-				{
-					"label": "Purple",
-					"value": 7
-				},
-				{
-					"label": "Orange",
-					"value": 8
-				}
-			]
+			"$import": "../0x0086/templates/aeotec_template.json#base_color_options",
+			"label": "LED Color: Light Sensor Report",
+			"defaultValue": 0
 		},
-		"20": {
-			"label": "Temperature Scale",
-			"description": "Report temperature in Celsius or Fahrenheit",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Celsius",
-					"value": 0
-				},
-				{
-					"label": "Fahrenheit",
-					"value": 1
-				}
-			]
+		"14": {
+			"$import": "../0x0086/templates/aeotec_template.json#celsius_fahrenheit_0",
+			"valueSize": 1
 		},
-		"21": {
-			"label": "Temperature Threshold Value to Report",
-			"description": "Minimum temperature change to report",
-			"unit": "0.1 degrees (C/F)",
+		"15": {
+			"label": "Temperature Change Threshold",
 			"valueSize": 2,
+			"unit": "0.1 °(C/F)",
 			"minValue": 0,
 			"maxValue": 250,
-			"defaultValue": 25
+			"defaultValue": 20,
+			"unsigned": true
 		},
-		"22": {
-			"label": "Light intensity Threshold Value to Report",
-			"description": "Minimum light intensity change to report",
-			"unit": "Lux",
+		"16": {
+			"label": "Light Intensity Change Threshold",
 			"valueSize": 2,
+			"unit": "lux",
 			"minValue": 0,
 			"maxValue": 10000,
-			"defaultValue": 100
+			"defaultValue": 100,
+			"unsigned": true
 		},
-		"23": {
-			"label": "Temperature Sensor Report Interval",
-			"description": "Interval for timed temperature reports",
-			"unit": "Seconds",
+		"17": {
+			"label": "Timed Temperature Report Interval",
 			"valueSize": 2,
-			"minValue": 0,
+			"unit": "seconds",
+			"minValue": 1,
 			"maxValue": 32767,
-			"defaultValue": 3600
+			"defaultValue": 3600,
+			"unsigned": true
 		},
-		"24": {
-			"label": "Light Sensor Report Interval",
-			"description": "Interval for timed light level reports",
-			"unit": "Seconds",
+		"18": {
+			"label": "Timed Light Sensor Report Interval",
```

Comment:
I think we should define somewhere how to call the automatic/timed/whatever reports, so we can make it consistent everywhere.